### PR TITLE
Added maintainers and code of conduct drafts

### PR DESCRIPTION
**Pull request by breakpointer**
# Description

Added files to inform the community of the maintainer group in the Baseline project and also the projects code of conduct. 

## Motivation and Context

It solves the problem of people not knowing how to become maintainers, what maintainers do, and what is expected of someone with "maintainer" status. 

## Type of change
Documentation

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+ - show commitment over time with multiple PRs merged
+ - be reliable in completing issues to which they have been assigned
+ - attend the weekly maintainer meetings (with occasional absences allowed)
+ - demonstrate competency in software development
+ - follow the project style and testing guidelines
+ - have a high degree of understanding of the project architecture
+ - be welcoming to others in the community who are using the project
+ - contribute in ways that substantially improve the quality of the project and the experience of people using it
+ - follow good branch naming conventions: [good references here](https://gist.github.com/digitaljhelms/4287848)
+ - Follow linting conventions for committing changes and by others
```
Comment (kthomas) [inline MAINTAINERS.md:58]: - follow the project style and linting conventions :D

good amount of irony here, so calling it out :)

File: MAINTAINERS.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+With either path you become a "provisional maintainer". As such you will need to show consistent contributions of code to the project. This can be in the form of pull requests that get merged into master. Or it can be in the form of system architecture and related artifacts that guide the development activities of others. 
+
+All provisional maintainers must meet with the existing maintainers and demonstrate they are capable of the following: 
+ - Running the project locally
+ - Using the testing framework
+ - Explaining the components of the system architecture
+ - Walking through the code and explain the baseline process
+
+Once the provisional maintainer demonstrates their capabilities they existing maintainers will vote during the next scheduled maintainer meeting to give the prospect full maintainer status. Members must vote with 2/3rds majority to add a new maintainer. Voting that results in a tie or potentially other issue will be brought to the TSC for review.
```
Comment (bitwiseguy) [inline MAINTAINERS.md:43]: "demonstrates their capabilities they existing" --> "demonstrates their capabilities, the existing"

File: MAINTAINERS.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+## Slack 
+Maintainers meet and discuss issues virtually via the private #maintainers slack room in the [baseline slack](ethereum-baseline.slack.com). 
+
+## Weekly Meetings
+There are weekly *Maintainers meetings* where members can discuss plans and issues related to the project, updates, release planning, and other related topics. These meetings are not public but special exceptions can be granted for members of the baseline community, experts, or other key participants. The summaries of the meetings will be posted to the public #maintainer-meeting slack room.
+
+# How to stop being a maintainer?
+
+If you are a maintainer you loose maintainer status if any of the following occur:
```
Comment (bitwiseguy) [inline MAINTAINERS.md:71]: "loose" --> "lose"

File: MAINTAINERS.md (hunk 1 above)
Comment (breakpointer) [reply MAINTAINERS.md:58]: Yeah, we haven't decided on those yet... so ¯\_(ツ)_/¯

File: MAINTAINERS.md (hunk 2 above)
Comment (breakpointer) [reply MAINTAINERS.md:43]: Fixed

File: MAINTAINERS.md (hunk 3 above)
Comment (breakpointer) [reply MAINTAINERS.md:71]: Fixed. Thanks!

**Pull request merged**